Corona-Warn-App/src/main/java/de/rki/coronawarnapp/ui/submission/SubmissionQRCodeScanFragment.kt

```diff
@@ -108,6 +95,7 @@ class SubmissionQRCodeScanFragment : Fragment() {
 
     override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
         super.onViewCreated(view, savedInstanceState)
+        binding.lifecycleOwner = this
```

Comment:
This is very repetitive in most Fragments and easily forgotten to be set. Can this be improved? Eg an abstract function that has to be implemented